src/app/organizations/organization/update-organization-description/update-organization-description.component.html

```diff
@@ -1,5 +1,5 @@
 <h1 mat-dialog-title>Edit {{ data.type }} description</h1>
-<div mat-dialog-content>
+<div mat-dialog-content id="updateOrganizationDescriptionWindow">
```

Comment:
use data-cy https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

Reply:
interesting thanks

Reply:
should i change all the other selectors in that file to data-cy instead of #id?

Reply:
Just yours is fine